src/components/ColumnTitle/ColumnTitle.tsx

```diff
@@ -24,12 +25,22 @@ export const ColumnTitle: FunctionComponent<ColumnTitle> = (props) => {
         return <div>{title}</div>
     }
 
-    const filterable = props.rowMeta.filterable
-    const sort = <ColumnSort
+    const noSortable = props.widgetMeta.type === FieldType.multivalue
```

Comment:
It's more readable to write this type of check with the help of Array.prototype.include:
```tsx
const noSortable = [
    FieldType.multivalue,
    FieldType.multifield,
    // ...
    FieldType.hint
].includes(props.widgetMeta.type)
```

Reply:
performed